app/controllers/api/v1/comments_controller.rb

```diff
@@ -0,0 +1,17 @@
+class Api::V1::CommentsController < Api::V1::ApplicationController
+  before_action :authenticate_user!
+  def index
+    comments = Comment.where(user_id: params[:user_id])
+    render json: comments, status: :ok
+  end
+
+  def create
+    comment = Comment.new(text: params[:comment][:text], post_id:, user_id:)
+
```

Comment:
- [ ] Where the value of post_id and user_id is read from? The both come from the URL params. Let's get them and fix this method.